packages/cart/src/models/line-item-adjustment-line.ts

```diff
@@ -0,0 +1,28 @@
+import { generateEntityId } from "@medusajs/utils"
+import {
+  BeforeCreate,
+  Entity,
+  ManyToOne,
+  OnInit
+} from "@mikro-orm/core"
+import AdjustmentLine from "./adjustment-line"
+import LineItem from "./line-item"
+
+@Entity({ tableName: "cart_line_item_adjustment_line" })
+export default class LineItemAdjustmentLine extends AdjustmentLine {
```

Comment:
suggestion: just to be extra safe that this works, would recommend running `yarn run migration:initial` to see if it produces the correct SQL.